### DISCUSSION TITLE
I have a problem With Copilot

### DISCUSSION THREAD

**Discussion by Andru0Gx** [community/community#43685] [Copilot Conversations]
I applied for GitHub Global Campus as a student and they accepted me since November 28, 2021 and all this time has been great but from today when I entered my account I realized that GitHub Copilot had been removed from me, I checked everything and I don't understand why they removed it, the strange thing is that I still have the PRO plan and all other benefits but I can no longer use the Copilot

![image](https://user-images.githubusercontent.com/95187524/211246428-93e36e83-12ab-47d7-b70d-a6a364150e42.png)

**Comment by radar-blip** [comment]
If you look through and/or read some of these recent discussions, it seems a lot of people are having issues with copilot today.
Hang in there!

**Reply by Andru0Gx** [reply]
Yeah bro I just saw it! Thank you for the information!

**Reply by radar-blip** [reply]
No problem. Good luck!
